mantidproject/mantid
# Increase BinWidth precision in Dynamic Kubo Toyabe Fit function

**Description of work.**
Changed the precision of the QtDoublePropertyManager so that the BinWidth for Dynamic Kubo Toyabe Algorithm will display to 3 d.p. rather than just two. Previously it would display to 2 d.p. which wasn't suitable, especially as minimum value is 0.001.


**To test:**
1. Open Muon Analysis GUI
2. Load a run
3. Open the fitting tab
4. Right click in empty Property/Value table to add a function
5. Search for DynamicKuboToyabe and select
6. The Bin Width should show as 0.050. 
7. Change the bin width to any 3 decimal number between 0.001 and 0.100 . It should continue to show as a number to 3 d.p.

Fixes #31890 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
